### PR TITLE
Adding a file

### PR DESCRIPTION
**Pull request by katt-y**
Bellman-Ford Algorithm Implementation.

### CODE REVIEW

File: README.md
Comment (JaydevSR) [inline README.md:25]: After this you can also add execution instructions if any (in your case it is just running `python filename.py`).

**TIP**: In a larger project a library will most probably export some functions for direct use of the users (for example scipy exports scipy.integrate etc.). In such case the documentation will consist of something like:

```
function(arg1, arg2, ...)

Performs/Calculates something and something.

More explaination ...

Examples:
    ```
    an example
    ```
```

This is basically all there is to documenting code for others.

File: README.md
Comment (JaydevSR) [inline README.md]:
```suggestion
Running the python file returns the Minimum Spanning Tree obtained by implementing the algorithm on a directed graph. In order calculate the Vertex distance from the Source vertex, just the add the weights of all the edges in the path from the source vertex to the Vertex.
```

Some minor fixes.

File: algorithms/bellman_ford.py
Comment (JaydevSR) [inline algorithms/bellman_ford.py]: Usual convention is to leave two lines before function/class definitions. Follow this in the whole file.

**TIP**: Following code convention is very important in a big project as multiple people may have different styles of writing code. Hence, it is important for everyone have consistent changes.

File: algorithms/bellman_ford.py
Comment (JaydevSR) [inline algorithms/bellman_ford.py]:
```suggestion
    def bellman_ford(self, j):
```

Convention!

Comment (JaydevSR) [reply algorithms/bellman_ford.py]: Follow this in the whole file.

File: algorithms/bellman_ford.py
Comment (JaydevSR) [inline algorithms/bellman_ford.py]: Usual python convention: **spaces around operators i.e. only one space before and only one space after**. This includes: +, -, /, *, =, ==.

This should be followed thoroughly.

File: algorithms/bellman_ford.py
Comment (JaydevSR) [inline algorithms/bellman_ford.py:53]: Remove unnecessary empty lines.

File: algorithms/bellman_ford.py
Comment (JaydevSR) [inline algorithms/bellman_ford.py]:
```suggestion
# Taking inputs for edges and vertices for graph in order to perform Bellman Ford.
```

File: algorithms/bellman_ford.py
Comment (JaydevSR) [inline algorithms/bellman_ford.py]:
```suggestion
d=input('The name of the vertices is numeric that is from 1 to n where n is the number of vertices.\nPress any key to continue : ')
```
As the line is too long!

File: algorithms/bellman_ford.py
Comment (JaydevSR) [inline algorithms/bellman_ford.py]:
```suggestion
    f= input('Now you have to input the details for different edges in the graph.\nPress any key to stop giving details (press c to continue): ')
```

Long line

**Review by JaydevSR — CHANGES_REQUESTED** [review]
Hi, @katt-y. Thanks for the contribution! This review contains some of the changes that are required for this PR to be merged. Nothing major just some conventions here and there (spaces around operators, unnecessary empty lines etc.).

**TIP**: This is what happens once you submit a PR. Even though you can get your code working, the maintainers of a library expect some extent of consistency in the codebase so they'll make you do a lot of edits (more extra works and delays unfortunately). But this is a very crucial step which make the whole open source idea work!

**Comment by JaydevSR** [comment]
**TIP**: If someone suggests some changes, then you can directly commit those from GitHub using the "Files Changed" tab where you can "Add suggestions to batch" and then commit! This really save a lot of time. Also, the small edits that I have requested can be directly from GitHub (using a commit named "suggested changes" or something like that). But be sure to sync your local repository after that in case you work again on adding something else! Because git conflicts is something you always want to avoid as it wastes a lot of time!

File: algorithms/bellman_ford.py
Comment (katt-y) [reply algorithms/bellman_ford.py:53]: I have committed to all the changes suggested by you, except this one change of removing empty lines. Can this be done on GitHub?

**Review by katt-y — COMMENTED** [review]
I have committed all the changes suggested except removing empty lines wherever possible.

Can this be done using Github?

**Comment by JaydevSR** [comment]
You can make small edits on github just by opening the file that you want to edit (in your forked repo) and clicking on the pencil button on the top right corner.

**Comment by JaydevSR** [comment]
Just address the things that I have reviewed about the extra empty lines and spaces around operators. After this I will merge the PR!

**Comment by katt-y** [comment]
I have committed to both changes.

Thanks!!

**Review by JaydevSR — APPROVED** [review]

**Comment by JaydevSR** [comment]
Great job! The PR looks clean and can be merged now! Congrats on becoming a pull shark (check your profile for the cool flair) 😄

**Comment by JaydevSR** [comment]
Also, as promised you'll get a treat from me, so catch me when I look like I am free! 🥇

**Comment by katt-y** [comment]
Thanks, bhaiya.

Woohoo!!

**Pull request merged**
